content/applications/finance/accounting/reporting/overview/customize.rst

```diff
@@ -1,90 +1,248 @@
+defined in another report.
+
+'Prefix of Account Codes' engine
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This engine is used to match amounts made on accounts using the prefixes of these accounts' codes as
+variables in an arithmetic expression.
+
+.. example::
+   | **21 + 10 - 5**
+   |  Adds the balances of the move lines made on accounts whose codes start with `21` and `10`, and
+     subtract the balance of the ones on the accounts with the prefix `5`.
+
+It is also possible to ignore a selection of sub-prefixes.
+
+.. example::
+   | **21 + 10\\(101, 102) - 5\\(57)**
+   | Does the same as the previous example, but ignoring the prefixes `101`, `102` and `57`.
+
+You can apply 'sub-filtering' on **credits and debits** using `C` and `D` suffixes. In this case, an
+account will only be considered if its prefix matches, *and* if the total balance of the move lines
+made on this account is **credit/debit**.
+
+.. example::
+   Account `210001` has a balance of -42 and account `210002` has a balance of 25. The formula
+   **21D** only matches the account `210002`, and hence return 25. `210001` is not matched, as its
+   balance is *credit*.
+   
+Prefix exclusions can be mixed with `C` and `D` suffixes.
+
+.. example::
+   **21D + 10\\(101, 102)C - 5\\(57)**
+
+To match the letter `C` or `D` in a prefix and not use it as suffix, use an *empty* exclusion.
+
+.. example::
+   | **21D\\()**
+   | Matches accounts whose code starts with `21D`, regardless of their balance sign.
+
+'External Value' engine
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The 'external value' engine is used to refer to **manual** and **carryover values**. Those values
+are not stored using `account.move.line`, but with `account.report.external.value`. Each of these
+objects directly points to the expression it impacts, so very little needs to be done about their
+selection here.
+
+**Formulas** can be one of the following:
+
+- **sum** : If the result must be the sum of all the external values in the period;
+
+- **most_recent**: If the result must be the value of the latest external value in the period.
+
+In addition, **subformulas** can be used in two ways:
+
+- **rounding=X** : Replacing **'X'** by a number, instructs to round the amount to X decimals;
+
+- **editable** : indicates this expression can be edited manually, triggering the display of an icon
+  in the report, allowing the user to perform this action.
+
+Both subformulas can be mixed, by separating them with a `;`.
+
+.. example::
+   **editable;rounding=2**
+    | Is a correct subformula mixing both behaviors.
+
+'Custom Python Function' engine
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This engine is a means for developers to introduce custom computation of expressions on a
+case-by-case basis. The formula is the name of a **python function** to call, and the subformula is
+a **key** to fetch in the **dictionary** returned by this function. Use it only if you are making a
+custom module of your own.
```

Comment:
I think this last sentence could be a note, but looks fine that way, too

Reply:
I'll leave it as is